### PR TITLE
Feature/shop profile

### PR DESCRIPTION
**Pull request by polig57**

### CODE REVIEW

**Comment by NitramSomar** [comment]
I will update the few minor changes we discussed.

File: client/src/component/Shop_Profile/Shop_Profile.jsx [hunk 1]
```diff
@@ -1,10 +1,42 @@
-import React from 'react'; 
+import React from 'react';
 import style from './Shop_Profile.module.css';
+import Bar from '../Navigation/Bar';
+import shops from './shop';
 
 
 const Shop_Profile = () => (
-
-    <div></div>
```
Comment (Emilio-miloCastaneda) [inline client/src/component/Shop_Profile/Shop_Profile.jsx:7]: what is going on in the shop profile component.

File: client/src/component/Shop_Profile/Shop_Profile.jsx [hunk 2]
```diff
@@ -1,10 +1,42 @@
...
+                    <div className={style.storeInfo}>
+                        <p className={style.name}>{shops[0].store}</p>
+                        <p className={style.loc}>{shops[0].location}</p>
+                        <p className={style.rating}>{shops[0].stars}</p>
+                    </div>
+                </div>
+            </div>
+            <div className={style.reviewContainer}>
+            <span className={style.reviews}>Reviews</span>
+                <>
```
Comment (Emilio-miloCastaneda) [inline client/src/component/Shop_Profile/Shop_Profile.jsx:22]: you have an empty tag here.

**Review by NitramSomar — APPROVED** [review]
Everything looks good I will make the few changes we discussed.

**Review by Emilio-miloCastaneda — APPROVED** [review]

**Pull request merged**